velox/docs/functions/spark/regexp.rst

```diff
@@ -34,3 +34,44 @@ See https://github.com/google/re2/wiki/Syntax for more information.
     the entire string by anchoring the pattern using ``^`` and ``$``. ::
 
         SELECT rlike('1a 2b 14m', '\d+b'); -- true
+
+.. spark:function:: regex_replace(string, pattern, overwrite) -> varchar
+
+    Replaces all substrings in ``string`` that match the regular expression ``pattern`` with the string ``overwrite``. If no match is found, the original string is returned as is.
+    There is a limit to the number of unique regexes to be compiled per function call (kMaxCompiledRegexes defined in velox/functions/lib/Re2Functions.h).
+
+    Parameters:
+
+    - **string**: The string to be searched.
+    - **pattern**: The regular expression pattern that is searched for in the string.
+    - **overwrite**: The string that replaces the substrings in ``string`` that match the ``pattern``.
+
+    Examples:
+
+    ::
+
+        SELECT regex_replace('Hello, World!', 'l', 'L'); -- 'HeLLo, WorLd!'
```

Comment:
Would it be possible to include examples with regular expressions? This example uses a simple string.

Reply:
Fixed